MaciASL.xcodeproj/project.pbxproj

```diff
@@ -458,6 +478,24 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		712536FB26F63EE900C5FC40 /* Set Sparkle entries */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Set Sparkle entries";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if [ \"$CONFIGURATION\" == \"Release\" ]; then\nplutil -insert \"SUAllowsAutomaticUpdates\" -bool true build/Release/MaciASL.app/Contents/Info.plist\nplutil -insert \"SUEnableAutomaticChecks\" -bool true build/Release/MaciASL.app/Contents/Info.plist\nplutil -insert \"SUFeedURL\" -string \"https://raw.githubusercontent.com/acidanthera/MaciASL/appcast/appcast.xml\" build/Release/MaciASL.app/Contents/Info.plist\nplutil -insert \"SUPublicEDKey\" -string \"d6TTl9OeRP+CH+IiQwgLCUsxj6DXK+Fnrcfmcj61ANk=\" build/Release/MaciASL.app/Contents/Info.plist\nplutil -insert \"SUScheduledCheckInterval\" -integer \"86400\" build/Release/MaciASL.app/Contents/Info.plist\nfi\n";
```

Comment:
- Is it `Release` or `RELEASE`?
- Can running build multiple times result in duplicates?
- Do not use relative paths like `build/Release`, there is `${TARGET_BUILD_DIR}` variable.

Reply:
> Is it Release or RELEASE?

In archiving script I saw it being `Release`, so I used that as well.

> Can running build multiple times result in duplicates?

No, I just tried it locally and it does output this error for example: `Build/Products/Release/MaciASL.app/Contents/Info.plist: Could not modify plist, error: Value true already exists at key path SUAllowsAutomaticUpdates`

> Do not use relative paths like build/Release, there is ${TARGET_BUILD_DIR} variable.

Thanks! I am still learning.